src/thunderbots/software/ai/navigator/obstacle/obstacle.h

```diff
@@ -0,0 +1,25 @@
+// An obstacle is an area to avoid based on the size of the robot and its velocity
```

Comment:
Should tweak this a bit to be more consistent with our other files, ie.
``` suggestion
/**
 * An obstacle is an area to avoid based on the size of the robot and its velocity
 */
```

Reply:
ok